data-raw/tranform_inputs_complete.R

```diff
@@ -885,9 +1018,11 @@ generate_wilkins_flow <- function(calsim_data, wilkins_node) {
 
 wilkins_flow_2008_2009 <- generate_wilkins_flow(calsim_2008_2009, wilkins_node)
 wilkins_flow_2019_biop_itp <- generate_wilkins_flow(calsim_2019_biop_itp, wilkins_node)
+wilkins_flow_run_of_river <- generate_wilkins_flow(calsim_run_of_river, wilkins_node)
```

Comment:
Warning message:
Using an external vector in selections was deprecated in tidyselect 1.1.0.
ℹ Please use `all_of()` or `any_of()` instead.
  # Was:
  data %>% select(vernalis_node)

  # Now:
  data %>% select(all_of(vernalis_node))

Getting this warning - I think just because of a tidyverse update but would be good to fix the select statements to stop this warning from coming up. See <https://tidyselect.r-lib.org/reference/faq-external-vector.html>.
This warning is displayed once every 8 hours.
Call `lifecycle::last_lifecycle_warnings()` to see where this warning was generated.

Reply:
I'm no longer getting this error, but agree that it is coming from an old version of Tidyverse. Select(all_of()) doesn't really make sense for Wilkins because you want to select multiple columns. I'll look into further.